filter/ArticleMedraXmlFilter.php

```diff
@@ -207,7 +207,35 @@ function createArticleNode(DOMDocument $doc, Submission|Galley $pubObject): DOME
         if (!empty($submissionGalleys)) {
             $this->appendTextMiningCollectionNodes($doc, $articleNode, $article, $submissionGalleys);
         }
-
+        
+        $issueId = $article->getCurrentPublication()->getData('issueId');
+        if ($cache->isCached('issues', $issueId)) {
+            $issue = $cache->get('issues', $issueId);
+        } else {
+            $issue = Repo::issue()->get($issueId, $context->getId());
+            if ($issue) $cache->add($issue, null);
+        }
+        $accessRights = null;
+        if($context->getData('publishingMode') == PUBLISHING_MODE_OPEN){
+                $accessRights = 'openAccess';
```

Comment:
indentation is not correct in many lines that were added here, so please try to correct them all

Reply:
Hi @bozana , done this adjustment. 
Thanks,
@njikonga-med